onnx/defs/operator_sets.h

```diff
@@ -1058,6 +1058,16 @@ class OpSet_Onnx_ver18 {
   }
 };
 
+// Forward declarations for ai.onnx version 19
+
+// Iterate over schema from ai.onnx version 19
+class OpSet_Onnx_ver19 {
+ public:
+  static void ForEachSchema(std::function<void(OpSchema&&)> fn) {
+    (void)fn;
```

Comment:
tiny nit: perhaps add a TODO saying this line should be removed after introducing an actual new schema?

Reply:
Done.